lib/Platform/Intl/java/com/facebook/hermes/intl/DateTimeFormat.java

```diff
@@ -244,9 +245,13 @@ private void initializeDateTimeFormat(List<String> locales, Map<String, Object>
     if (JSObjects.isUndefined(timeZone)) {
       timeZone = DefaultTimeZone();
     } else {
-      String normalizedTimeZone = normalizeTimeZoneName(JSObjects.getJavaString(timeZone));
-      if (!isValidTimeZoneName(normalizedTimeZone)) {
-        throw new JSRangeErrorException("Invalid timezone name!");
+      try {
+        if (!isValidTimeZoneName(timeZone.toString())) {
```

Comment:
It seems like we don't need to perform this validation any more, since `normalizeTimeZoneName` will already throw an exception for any timezone that is invalid.